testing/matrix-sdk-ffi-macros/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+description = "Helper macros to write FFI bindings"
+edition = "2021"
```

Comment:
2024?

Reply:
![image](https://github.com/user-attachments/assets/b5db9b60-940b-4845-abd0-9fd9abce254a)

Reply:
@bmarty You can learn what a Rust edition is by reading https://doc.rust-lang.org/edition-guide/editions/index.html :-).